central/scannerdefinitions/handler/handler_test.go

```diff
@@ -63,8 +63,8 @@ func (s *handlerTestSuite) SetupTest() {
 func (s *handlerTestSuite) TearDownSuite() {
 	entries, err := os.ReadDir(s.tmpDir)
 	s.NoError(err)
-	s.LessOrEqual(len(entries), 1)
-	if len(entries) == 1 {
+	s.LessOrEqual(len(entries), 2)
+	if len(entries) == 2 {
 		s.True(strings.HasPrefix(entries[0].Name(), definitionsBaseDir))
 	}
```

Comment:
If another entry is expected, probably should add another assertion.

```suggestion
	s.LessOrEqual(len(entries), 2)
	if len(entries) == 2 {
		s.True(strings.HasPrefix(entries[0].Name(), definitionsBaseDir))
		s.True(strings.HasPrefix(entries[1].Name(), <something>))
	}
```